src/winter_onboarding_2021/fleet_management_service/routes.clj

```diff
@@ -15,15 +16,27 @@
                                  (:title data)
                                  (:content data)))))))
 
+(defn protect [handler allowed-roles]
```

Comment:
`protect` doesn't indicate what it is doing. Rename this to something that does. eg. `authentication-required`